Entities/ACrab.cpp

```diff
@@ -2172,35 +2178,32 @@ void ACrab::Update()
             m_pJetpack->AlarmOnEmit(m_Team);
             // Deduct from the jetpack time
             m_JetTimeLeft -= g_TimerMan.GetDeltaTimeMS();
+			if (m_JetTimeLeft < 0) { m_JetTimeLeft = 0; }
```

Comment:
Can probably streamline this - `m_JetTimeLeft = std::max(0, m_JetTimeLeft - g_TimerMan.GetDeltaTimeMS())` and then I guess you don't need the check below when not jumping?